pkg/agent/flowexporter/connections/connections.go

```diff
@@ -15,194 +15,49 @@
 package connections
 
 import (
-	"encoding/binary"
 	"fmt"
 	"sync"
-	"time"
 
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/klog"
 
 	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
-	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter/flowrecords"
 	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
 	"github.com/vmware-tanzu/antrea/pkg/agent/metrics"
-	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
 	"github.com/vmware-tanzu/antrea/pkg/agent/proxy"
-	"github.com/vmware-tanzu/antrea/pkg/querier"
 )
 
-var serviceProtocolMap = map[uint8]corev1.Protocol{
-	6:   corev1.ProtocolTCP,
-	17:  corev1.ProtocolUDP,
-	132: corev1.ProtocolSCTP,
-}
-
 type ConnectionStore interface {
```

Comment:
Is this interface needed? Are we using the mock of this interface for testing somewhere?

Reply:
I was thinking about using this instead of denyConnStore and can better present the methods of connectionStore struct. It's actually not needed that much. Removed it.